### ISSUE TITLE
Add setting to pick what types of elements should be replaced

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Woundorf/foxreplace#56]
```
What steps will reproduce the problem?
1. pick a word that is in a javascript generated combo box on a webpage
2. set a replace option that targets that word
3. press F2

What is the expected output? What do you see instead?
the text in the combo box populated via AJAX/Javascript selections should 
remain untouched

What version of the product are you using? On what operating system?
0.12.2 Windows XPSP3

Please provide any additional information below.
basically page data should remain untouched
```

Original issue reported on code.google.com by `DavidTaw...@gmail.com` on 16 Sep 2011 at 7:13

**Comment by bluebear94** [comment]
Any updates?

**Comment by Woundorf** [comment]
Not yet, sorry.

**Comment by jkunimune** [comment]
It would actually be especially helpful to have the option to ignore <style> elements.  I want to have a replacement from /tion\b/ to /cion/, but this breaks `position:` attributes in <style> blocks which changes the page layout.  Since there's already options to exclude URLs and scripts, I imagine this would be similar.
